packages/elements/src/components/ino-switch/ino-switch.scss

```diff
@@ -7,35 +7,39 @@
 $n-5: #abb1cc;
 $n-6: #cbd0e0;
 
-@mixin set-track-border-color($color) {
+/* colors */
+$primary: #2C02FF;
+$focus: #2D5DFF4D;
+
+@mixin set-track-border-color {
   .mdc-switch__track {
     border: 1.5px solid white;
     margin: -1.5px;
-    box-shadow: 0 0 0 5px $color;
+    box-shadow: 0 0 0 5px $focus;
   }
 }
 
 @each $color in theme.$color-schemes {
```

Comment:
We can remove this selector because we don't support all the theme colors anymore